Lib/queue.py

```diff
@@ -25,6 +25,11 @@ class Full(Exception):
     pass
 
 
+class Cancelled(Exception):
+    'Exception raised by Queue.get(block=True) when cancelled'
```

Comment:
The `pass` is redundant and docstrings should be triple quoted:)
```suggestion
class Cancelled(Exception):
    '''Exception raised by Queue.get(block=True) when cancelled'''
```

Reply:
Indeed, I simply wanted to keep the same style with which the `Full` exception is written, just a couple of lines above.

https://github.com/python/cpython/blob/f2e5a6ee628502d307a97f587788d7022a200229/Lib/queue.py#L23-L25

Question:

  - Shall we add also the same changes to `Full`?

Reply:
```suggestion
class Cancelled(Exception):
    """Exception raised by Queue.get(block=True) when cancelled."""
```

Furthermore, while we're at it, per PEP 257 (PEP-257) and widespread convention, docstrings must use triple double quotes and summary lines should be complete sentences that always end with a period.